### ISSUE TITLE
anystyle train crashing

### ISSUE THREAD

**Issue by lr210** [inukshuk/anystyle-cli#13]
anystyle train crashes when I try to train with a particular xml file. When I run anystyle check it appears as if the file is valid.

The error is as follows: 
ruby: vmath.c:281: xvm_expma: Assertion `r != NULL && ((uintptr_t)r % 16) == 0' failed.

I have replicated this on two different machines: a windows machine and a linux machine.
![anystyle](https://user-images.githubusercontent.com/60696915/125493948-97259fc8-5182-4ca1-be19-5a5804c68b47.PNG)

**Comment by inukshuk** [comment]
I'd check for any obvious errors in the training data, such as blank nodes. I assume that there are only 5 labels in the training set is intentional? If you don't find anything, I'm happy to take a quick look at the data if you can post it somewhere.

**Comment by lr210** [comment]
The issue was due to blank nodes. Thanks for your help! Would it be possible to produce a nice error when running anystyle check when there are blank nodes?
